test/testsuite.sh

```diff
@@ -28,15 +28,28 @@ echo "        TOMCAT_CYCLE_COUNT=$TOMCAT_CYCLE_COUNT"
 echo "        ITERATION_COUNT=$ITERATION_COUNT"
 echo "        IMG=$IMG"
 echo "        HTTPD_IMG=$HTTPD_IMG"
+if [ ! -z ${MPC_CONF+x} ]; then
+echo "        MPC_CONF=$MPC_CONF"
```

Comment:
I don't want to be nitpicking, but the echo would be better indented.

Reply:
fixed.